.goreleaser.yml

```diff
@@ -29,7 +29,6 @@ builds:
   goos:
     - linux
   goarch:
-    - amd64
```

Comment:
Why was this removed?

Reply:
this arch will be generated by the SLSA compliant builder, so we don't want goreleaser to build it too and overwrite the SLSA build in the release assets.